crates/pathfinder/src/core.rs

```diff
@@ -10,6 +10,10 @@ use web3::types::{H128, H160, H256};
 #[derive(Debug, Copy, Clone, PartialEq, Eq, Hash, Deserialize, Serialize, PartialOrd, Ord)]
 pub struct ContractAddress(pub StarkHash);
 
+/// The hash of a StarkNet class.
+#[derive(Debug, Copy, Clone, PartialEq, Eq, Hash, Deserialize, Serialize, PartialOrd, Ord)]
+pub struct ClassHash(pub StarkHash);
+
```

Comment:
I think this is the same as `ContractHash`? We should consider renaming things to be align with the new class concepts.

Reply:
> constructor arguments

Where do you see this? Entry point is not an argument.

Reply:
This PR is already 10 files so I'd go with type alias or something, with a todo to rename all...?